README.md

```diff
@@ -519,6 +519,29 @@ However, if you want to specify it in each example, you can do so
 let(:module_path) { '/path/to/your/module/dir' }
 ```
 
+#### Specifying trusted certificate extensions
+
+In Puppet 4 the trusted facts hash will have the standard keys (certname, domain, and hostname) populated
+based on the node name (as set with `:node`).
+
+By default, the test environment contains no custom trusted extensions. If you need to test against specific
+certificate extensions you can set those with a hash
+
+```ruby
+let(:trusted_fact_extensions) { {'pp_uuid' => 'ED803750-E3C7-44F5-BB08-41A04433FE2E', '1.3.6.1.4.1.34380.1.2.1' => 'ssl-termination'} }
```

Comment:
Would you be open to changing this to just `trusted_facts`? I see no reason to concern users with the implementation details.